docs/src/theme/Faq.js

```diff
@@ -1,19 +1,22 @@
 import React, { useState } from 'react'
 import cn from 'classnames'
 import styles from './faq.module.css'
+import genStyle from './faq.gen.module.css'
 
 const Question = ({ children, tags }) => (
-  <div className={cn(tags)}>{children}</div>
+  <div className={cn(genStyle.question, ...tags.map((tag) => genStyle[tag]))}>
+    {children}
+  </div>
 )
 
 const TagButton = ({ tag, isSelected, children, toggleSelected }) => (
   <li
     className={cn(
-      { [styles.selected]: isSelected },
-      tag + '_src-theme-',
-      'pills',
-      'pills__item',
-      { 'styles.pills__item--active': isSelected }
+      { [genStyle.selected]: isSelected },
+      genStyle[tag],
+      styles.pills,
+      styles.pills__item,
```

Comment:
@k9ert what are these classes doing in the first place? We figured out that they where not applied before, and nothing changed when we did apply them :sweat_smile:

Reply:
Nevermind, it does look better now.